components/tools/OmeroWeb/omeroweb/decorators.py

```diff
@@ -421,7 +421,7 @@ def get_authenticated_connection(self, server_id, request):
             return None
 
         session['connector'] = connector
-        return connection
+        return
```

Comment:
What does it mean for this return value to be `None`?

Reply:
Returning None from ```get_authenticated_connection()``` simply means we are not logged in.
I made this change because ```PyCharm``` told me that this was an "Unresolved reference", and inspection of the code confirms that if you reach this point, ```connection``` would not be defined.

Reply:
Cheers. Comparing with the previous block, I partially wonder if this should be `del session['connector']` then to match the `return None` behavior. Since (I think) `connector` must be `None` at this point, it likely doesn't matter too much. Which is all a long way of saying that at some point we should probably clean-up / simplify this block.

Reply:
Just for consistency with the style of the other conditions and the expected return value let's `return None` here or switch the other locations to just `return`.